gpkit/_cvxopt.py

```diff
@@ -1,4 +1,4 @@
-def cvxoptimize_fn(options):
+def cvxoptimize_fn(options={}):
```

Comment:
Empty dicts and lists are dangerous default arguments. I see that it probably doesn't matter here though.